### PR TITLE
Added changelog for 0.3.0

### PR DESCRIPTION
**Pull request by Nyholm**
We should release 0.3.0.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+### Added
+
+- Return types.
+- Many `InvalidArgumentException`s are thrown when you use invalid arguments. 
+- Integration tests for `UploadedFile` and `ServerRequest`.
+
+### Changed
+
+- We dropped PHP7.0 support. 
+- PSR-17 factories has been marked as internal. They do not fall under or BC promise until PSR-17 is accepted.
```
Comment (Zegnat) [inline CHANGELOG.md]: Should be “under _our_ BC”.
(Is it “factories has been” or “factories have been”? That always gets me.)

**Comment by Zegnat** [comment]
All the changes looked good to me, so no complaints on making them accessible as a new release. I’m in the release and iterate often camp anyway, and we can always make patch releases if we bump into anything.

**Comment by Nyholm** [comment]
Thank you for the review

File: CHANGELOG.md (hunk 1 above)
Comment (Nyholm) [reply CHANGELOG.md]: Thank you

**Pull request merged**